data/human/news.txt

```diff
@@ -72,6 +72,11 @@ news "merchant anywhere"
 			"Having both feet firmly on the ground doesn't make you take one step to completing your missions."
 			"The darkest hour is just before the departure."
 			"Don't look a gift shuttle in the engine outlets."
+			"Getting there is half the fun. Getting paid is the other half."
+			"Being a pilot is not all fun and games. We have angry customers, hostile pirates and asteroids to deal with."
+			"We are always a risk of being killed by faulty equipment, solar storms, asteroids, pirates or by angry customers. Ever wonder why you didn't just take a desk job?"
+			"Did you ever look at ships going into hyperspace? They appear to be moving backwards because they are moving faster than light."
```

Comment:
This feels like a pretty bold lore statement. Does it match existing lore?

Reply:
I'll move it to a philosopher section.